Tools/parametric_model/configs/fixedwing_longitudinal_model.yaml

```diff
@@ -0,0 +1,164 @@
+# general information
+model_name: "Gazebo Standard Plane"
+model_type: "Standard Plane Longitudinal Model"
+model_class:
+  "SimpleFixedWingModel"
+
+  # all vectors in FRD body frame if not specified otherwise
+model_config:
+  mass: 20.0
+  moment_of_inertia:
+    Ixx: 25.0
+    Iyy: 20.0
+    Izz: 20.0
```

Comment:
Compared to the mass, the moment of inertia seems quite high. Does this come from some reference model? If so, can you cite the source?

Reply:
Same as for the mass, these values were basically randomly selected, but should only result in a scaling of the result as well. Setting them all to 1.0 and using the identified coefficient led to exactly the same fit as before. But they should definitely be adapted to a value that is more consistent with the test log.